frontend/webservice/provisioning.cgi

```diff
@@ -473,7 +457,7 @@ sub _send_mpls_add_command {
     my $circuit_id = $args{'circuit_id'};
     warn "_send_mpls_add_command: Calling addVlan on circuit $circuit_id";
     my $cv = AnyEvent->condvar;
-    $client->addVlan(circuit_id => int($circuit_id), async => 1, async_callback => sub { 
+    $client->addVlan(circuit_id => int($circuit_id), async_callback => sub {
```

Comment:
Whatever the CGI's return is the http response. We have to block in the CGI  to preserve the http request response mapping. The async keyword causes an rabbitmq parameter check failure. So these have to be removed wherever they exist.